plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/order-by-control/custom-query-order-by-control.tsx

```diff
@@ -90,33 +85,24 @@ const OrderByControl = ( props: QueryControlProps ) => {
 	}
 
 	return (
-		<ToolsPanelItem
-			label={ __( 'Order by', 'woocommerce' ) }
+		<OrderByControl
+			selectedValue={ orderValue }
 			hasValue={ () =>
 				order !== DEFAULT_QUERY.order ||
 				orderBy !== DEFAULT_QUERY.orderBy
 			}
-			isShownByDefault
+			orderOptions={ orderOptions }
```

Comment:
I think we should add a help text in this case as well to somehow explain to users this one is going to affect only this specific block. If help text in default order is `This setting is sincyed across all Product Collection blocks using the default query type.`, maybe something like:

```suggestion
			orderOptions={ orderOptions }
			help={ __( 'Set the products order in this collection.' ) }
```

Reply:
Done in 7005065a99426172992efe83b6a19c8c5d0bbf07.